site/content/en/docs/FAQ/_index.md

```diff
@@ -128,6 +128,14 @@ Each `GameServer` inherits the IP Address of the Node on which it resides. If it
 the Node (which it should if it's a publicly addressable Node), that it utilised, otherwise it falls back to using the
 `InternalIP` address. 
 
+### How do I use the DNS name of the Node?
```

Comment:
Can we wrap this in a feature shortcode please, so it won't show in current documentation when we merge it:
See: https://agones.dev/site/docs/contribute/#within-a-page